Event/OAuthEvent.php

```diff
@@ -37,7 +37,9 @@ class OAuthEvent extends Event
     private $isAuthorizedClient;
 
     /**
-     * @param UserInterface $user
+     * @param UserInterface   $user
+     * @param ClientInterface $client
+     * @param Boolean         $isAuthorizedClient
```

Comment:
Done.
I just used the same CS which is already used in `OAuthEvent.php` and [ClientInterface.php](https://github.com/FriendsOfSymfony/FOSOAuthServerBundle/blob/master/Model/ClientInterface.php#L35) files.